FWCore/Framework/src/EventSelector.cc

```diff
@@ -409,41 +389,39 @@ namespace edm
   // Indicate if any bit in the trigger results matches the desired value
   // at that position, based on the Bits array.  If s is Exception, this
   // looks for a Exceptionmatch; otherwise, true-->Pass, false-->Fail.
-  bool 
-  EventSelector::acceptOneBit(Bits const& b, 
-    		       	       HLTGlobalStatus const& tr, 
+  bool
+  EventSelector::acceptOneBit(Bits const& b,
+    		       	       HLTGlobalStatus const& tr,
     		               hlt::HLTState const& s) const
   {
     bool lookForException = (s == hlt::Exception);
     Bits::const_iterator i(b.begin());
     Bits::const_iterator e(b.end());
```

Comment:
Chris,
    Just an oversight. I will remove them when you let me know that you have no other suggested changes, or in a half hour or so if I don't hear from you.